testData/preferences/jaws.json5

```diff
@@ -140,12 +139,12 @@
                         "OSM.UnderlineProximity": 5,
 
                         "Touch.FlickVelocityMin": 99,
-                        "Touch.TapEventDurationMax": 99,
-                        "Touch.TapInterEventDurationMax": 99,
-                        "Touch.TapTranslationMax": 99
-                    },
-                    "http://registry.gpii.net/applications/com.freedomscientific.jaws/enabled": true
-                }
+                        "Touch.TapEventDurationMax": 170,
+                        "Touch.TapInterEventDurationMax": 300,
+                        "Touch.TapTranslationMax": 50
+                    }
+                },
+                "http://registry.gpii.net/applications/com.freedomscientific.jaws/enabled": true
```

Comment:
Can this just be expressed as `enabled` in the above block, or does this type of parameter have to be passed as a separate URL?